homework-submission/week3/client/src/components/HomeworkWeek3/style.css

```diff
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+}
+
+ul {
+  margin: 0 auto;
+  padding: 10px 15px;
+  text-align: left;
+  background: rgba(255, 255, 255, 0.1);
+  display: table;
```

Comment:
`display: table`? We're not really using table layout here, flexbox would work fine in this case.

Reply:
Done :pray: